src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -725,13 +742,16 @@ const TwoWindingsTransformerCreationDialog = ({
     const handleSave = () => {
         setCreationError();
         let isFormValid = true;
+        let tabWithError = [];
         if (!characteristicsInputForm.validate()) {
             isFormValid = false;
+            tabWithError.push(0);
```

Comment:
I would use constants like:
	const CHARACTERISTICS_TAB = 0;
	const RATIO_TAP_TAB = 1;
	const PHASE_TAP_TAB = 2;
and use them everywhere in the code rather than 0/1/2
(will ease if we swap tabs or insert a new one)